lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -25,6 +25,20 @@
       - ansible
   retries: 5
 
+- name: Copy ssh keepAlive configs
+  copy: src={{ item.src }} dest={{ item.dest }}
+  become: yes
+  with_items:
+    - {src: 'sshd_config', dest: '/etc/ssh/sshd_config'}
+  when: preburn
+
+- name: Restart service ssh
+  become: yes
+  service:
+    name: ssh
+    state: restarted
+  when: preburn
```

Comment:
are we building a new VM with packer to reflect these changes?

Reply:
We are going to have to in order to reflect those changes, but for now let's just ship it.